browser/ui/views/tabs/brave_tab.cc

```diff
@@ -48,3 +49,11 @@ void BraveTab::ActiveStateChanged() {
   // https://github.com/brave/brave-browser/issues/23476/
   alert_indicator_button_->UpdateEnabledForMuteToggle();
 }
+
+absl::optional<SkColor> BraveTab::GetGroupColor() const {
+  if (tabs::features::ShouldShowVerticalTabs())
+    return {};
```

Comment:
Is this equivalent to returning `absl::nullopt`? Cool!

Reply:
Yes, it is. More precisely, it constructs an instance of any class by calling a constructor with empty param. https://en.cppreference.com/w/cpp/language/list_initialization  Copy-list-initialization - (8)

```
8) in a return statement with braced-init-list used as the return expression and list-initialization initializes the returned object
```